crypto/fipsmodule/rand/entropy/tree_drbg_jitter_entropy.c

```diff
@@ -151,7 +151,7 @@ static void tree_jitter_drbg_maybe_get_pred_resistance(
 
   if (tree_jitter_drbg->is_global == 0 &&
       tree_jitter_drbg->ube_protection != 1) {
-    CRYPTO_sysrand_for_seed(pred_resistance, RAND_PRED_RESISTANCE_LEN);
+    CRYPTO_sysrand(pred_resistance, RAND_PRED_RESISTANCE_LEN);
```

Comment:
what's the motivation for this change?

Reply:
Because `CRYPTO_sysrand_for_seed` is now equivalent to `CRYPTO_sysrand` and I didn't want to factor that in to the PRNG model. Later, I plan on removing `CRYPTO_sysrand_for_seed` completely.